papis/commands/open.py

```diff
@@ -33,36 +33,36 @@
     type: book
     year: '2009'
 
-This book has defined two marks. Each mark has a name and a value.
-If you tell the ``open`` command to open marks, then it will look for
-the marks and open the value (page number). This is the default behavior.
-However, if you go to the :ref:`configuration <marks-options>`
-you'll see that you can change the convention to what suits you.
+This book has defined two marks. Each mark has a name and a value. If you tell
+the ``open`` command to open marks, it will look for the marks and open the
+value (page number). This is the default behavior. However, if you go to the
+:ref:`configuration <marks-options>`, you'll see that you can change the
+convention to what suits you.
```

Comment:
```
This book has two defined marks. Each mark has a name and a value. If you tell
the ``open`` command to open marks, it will look for the marks and open the
value (page number). This is the default behavior. However, if you go to the
:ref:`configuration <marks-options>`, you'll see that you can change the
convention to what suits you.
```